core/tests/gae_suite.py

```diff
@@ -72,6 +78,10 @@ def main(args=None):
             raise Exception('Directory %s does not exist.' % directory)
         sys.path.insert(0, directory)
 
+    # Remove coverage from path since it causes conflicts with the standard
+    # Python html library.
```

Comment:
I don't understand this. Why would it cause conflicts (how are the two libs related)? Maybe add a link to a doc/reference, and/or explain in more detail?

Reply:
Added more elaborate comment.